lib/ansible/modules/files/file.py

```diff
@@ -79,7 +79,7 @@
 '''
 
 EXAMPLES = '''
-# change file ownership, group and mode. When specifying mode using octal numbers, first digit should always be 0.
+# change file ownership, group and mode. When specifying mode using octal numbers, add a leading 0
```

Comment:
The idea was to put `0` reference just above `mode` argument:
```yaml
  # here:
  mode: 0644
```

Reply:
That makes a *lot* more sense to me than what I was thinking you wanted. Thank you. I've made that change.